atk/Object.custom

```diff
@@ -88,8 +89,10 @@
 		}
 
 		[GLib.DefaultSignalHandler (Type=typeof(Atk.Object), ConnectionMethod="OverrideRefChild")]
-		protected virtual Atk.Object OnRefChild (int i) {
-			return null;
+		protected virtual Atk.Object OnRefChild (int i)
+		{
+			IntPtr raw = atksharp_object_base_ref_child (Handle, i);
+			return GLib.Object.GetObject (raw, true) as Atk.Object;
```

Comment:
Is this one of the methods which *may not* be implemented in the base class, which means we could get `IntPtr.Zero` as raw? If so, is it safe to call GetObject with a null pointer?